Bootstrap/Update.php

```diff
@@ -427,4 +428,42 @@ private function addConnectItemsIndex()
             }
         }
     }
+
+    /**
+     * Create the mapping table between connect remote categories and local categories.
+     */
+    private function createRemoteToLocalCategoriesTable()
+    {
+        if (version_compare($this->version, '1.1.2', '<=')) {
+            try {
+                $this->db->query('CREATE TABLE IF NOT EXISTS `s_plugin_connect_categories_to_local_categories` (
+                  `remote_category_id` int(11) NOT NULL,
+                  `local_category_id` int(11) unsigned NOT NULL,
+                  PRIMARY KEY (`remote_category_id`, `local_category_id`),
+                  CONSTRAINT s_plugin_connect_remote_categories_fk_remote_category_id FOREIGN KEY (remote_category_id) REFERENCES s_plugin_connect_categories (id) ON DELETE CASCADE,
+                  CONSTRAINT s_plugin_connect_remote_categories_fk_local_category_id FOREIGN KEY (local_category_id) REFERENCES s_categories (id) ON DELETE CASCADE
+                  ) ENGINE = InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;'
+                );
+                $result = $this->db->query('SELECT pcc.id, pcc.local_category_id
+                                FROM s_plugin_connect_categories pcc
+                                WHERE pcc.local_category_id is not null');
+
+                while ($row=$result->fetch()) {
```

Comment:
Missing one space between ```=```
Maybe you need to run ```bash .githooks/install-hooks.sh``` locally. After that before every commit php cs fixer will scan your code and notify you have some places where code doesn't have correct code style.

Reply:
the cs fixer ran but i wil fix this